src/messages.js

```diff
@@ -0,0 +1,9 @@
+var req = require.context('../lang', true, /\.json.*$/);  
+var exports = {};
+
+req.keys().forEach(function (file) {
```

Comment:
Just checked the docs, it relies on Object.keys, which is supported in IE9.

`Note: keys depends on Object.keys. You may need to polyfill it for older browsers.`
https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/Object/keys